full_name.rb

```diff
@@ -0,0 +1,19 @@
+# Write a program which asks for a person's first name, then middle, then last.  It should store each of these parts in an array. Finally, it should greet the person using their full name.  Call the program full_name.rb.
+
+def get_name
+    full_name = []
+    puts "What is your first name? "
+    first = gets.chomp
+    puts "What is your middle name? "
+    middle = gets.chomp
+    puts "What is your last name? "
+    last = gets.chomp
+    full_name.push(first, middle, last)
```

Comment:
This does fulfill the requirements of the assignment, but what it _actually_ wants you to do is populate the array "as you go" so that you don't have to store local variables. Also, you don't need the spaces after the question marks because `puts` adds a newline anyway.

```suggestion
    full_name = []
    puts "What is your first name?"
    full_name.push(gets.chomp)
    puts "What is your middle name?"
    full_name.push(gets.chomp)
    puts "What is your last name?"
    full_name.push(gets.chomp)
```